client/src/components/DataFiles/DataFiles.jsx

```diff
@@ -87,9 +88,21 @@ const DataFiles = () => {
     (state) => state.workbench.config.noPHISystem
   );
 
+  const authenticatedUser = useSelector(
+    (state) => state.authenticatedUser.user.username
+  );
+  const systemSplitArr = listingParams.system.split('.');
```

Comment:
Unfortunately we won't be able to get the project id using a split on periods, since some projects could have a namespace including a period. For example, locally, a project id would be `LOCAL.CEP-1234`